sdk/storage/azure-storage-blob/azure/storage/blob/_container_client.py

```diff
@@ -1091,7 +1091,7 @@ def upload_blob(
 
     @distributed_trace
     def delete_blob(
-            self, blob,  # type: Union[str, BlobProperties]
+            self, blob,  # type: str
```

Comment:
```suggestion
            self, blob: str,
```
I know it's not really in the scope of the changes..... but..... can't help it......